tests/test_geometry_challenge.py

```diff
@@ -11,14 +11,15 @@
 
 # Test cases as (alpha_degrees, expected_r) pairs
 TEST_CASES = [
+    (128.25, 2.18),
     (90.00, 0.00),
     (100.00, 0.29),
     (110.00, 0.80),
     (120.00, 1.50),
     (130.00, 2.33),
     (140.00, 3.25),
-    (150.00, 4.19),
-    (160.00, 5.05),
+    (150.00, 4.18),
+    (160.00, 5.04),
```

Comment:
Thanks for adjusting those values, they look legit. I had actually just extracted the original values from the visual construction and might have been a bit off for those two.